packages/xrpl-crypto/src/sha512/index.ts

```diff
@@ -0,0 +1,5 @@
+import { createHash } from 'crypto'
```

Comment:
With node 14 support dropped does it make sense to have a browser and node version?

Reply:
This is using the native node `crypto` module when in a node environment for performance reasons.